src/json.human.js

```diff
@@ -103,12 +112,15 @@
         }
     }
 
-    function _format(data) {
+    function _format(data, options, parentKey) {
+
         var result, container, key, keyNode, valNode, len, childs, tr,
             isEmpty = true,
             accum = [],
             type = getType(data);
 
+        var indexOf = [].indexOf || function(item) { for (var i = 0, l = this.length; i < l; i++) { if (i in this && this[i] === item) return i; } return -1; };
```

Comment:
can you move indexOf pollyfill to the module level so it's not created on each _format call?